cfn/paws-collector.template

```diff
@@ -514,14 +651,21 @@
                         {"Ref": "AWS::StackName"}
                     ]
                 ]},
-                "Type": "String",
-                "Tier": {"Ref": "PawsSecretParamTier"},
-                "Value": {
-                    "Fn::GetAtt": ["EncryptPawsSecret", "EncryptedText"]
+                "KeyId": {
+                    "Fn::GetAtt":["LambdaKmsKey","Arn"]
                 },
-                "Tags": {
-                    "StackId": {"Ref": "AWS::StackId"}
-                }
+                "Plaintext": {
+                    "Ref": "PawsSecret"
+                },
+                "Tier": {
+                    "Ref": "PawsSecretParamTier"
+                },
+                "Tags": [{
+                        "Key": "StackId",
+                        "Value": {
+                            "Ref": "AWS::StackId"
+                        }
```

Comment:
cosmetic: could you please format this using standard JSON formatting? e.g.
```json
{
    "Tags": [
        {
            "Key": "StackId",
            "Value": {
                "Ref": "AWS::StackId"
            }
        }
    ]
}
```

Reply:
sure